include/ExpressionParser.cc

```diff
@@ -0,0 +1,64 @@
+#include <vector>
+#include <string>
+#include <stack>
+#include <iostream>
+#include "utils.h"
+
+class ExpressionParser
```

Comment:
What's the purpose of this class?

Reply:
This class is to parse the matcher expression passed in the model conf file